helm/botkube/values.yaml

```diff
@@ -47,6 +47,8 @@ rbac:
       resources: ["*"]
       verbs: ["get", "watch", "list"]
   staticGroupName: &static-group-name "botkube-plugins-default"
+  # default channel in slack & discord is called "general"
+  channelGroupName: &channel-group-name "general"
```

Comment:
This is just for testing purposes, right? We shouldn't expose such option as it is up to the user to create proper resources. Of course we should document how to create (Cluster)Role + (Cluster)RoleBinding manually as a part of docs.